alerting/src/main/kotlin/org/opensearch/alerting/action/GetAlertsRequest.kt

```diff
@@ -11,26 +11,30 @@ import org.opensearch.alerting.model.Table
 import org.opensearch.common.io.stream.StreamInput
 import org.opensearch.common.io.stream.StreamOutput
 import java.io.IOException
+import java.util.Collections
 
 class GetAlertsRequest : ActionRequest {
     val table: Table
     val severityLevel: String
     val alertState: String
     val monitorId: String?
     val alertIndex: String?
+    val monitorIds: List<String>?
```

Comment:
Can we not add this and make `monitorId` a list? At the rest layer, we can detect if its a string or a list and pass it in the request as such. This allows there to not be any confusion of the `monitorId` and `monitorIds` field

Reply:
Alternatively, the current way may be better if we want a deprecation path to change the parameter for the API. For example, in the beginning we'd support both `monitorId` and `monitorIds` just so there is no breaking changes but in a major version we can remove `monitorId` so that even a single monitor is passed through `monitorIds` (since making `monitorId` a list is confusing as it implies there is only one).

Reply:
agreed will make the change.